libs/frontend/util-edit-text/src/lib/keymap.ts

```diff
@@ -109,5 +110,10 @@ export function buildKeymap(schema: Schema, overrideKeys?: { [key: string]: Comm
     })
   }
 
+  if ((type = schema.nodes.text)) {
+    bind('Space', textToLinkOnSpace(schema))
+    bind('Enter', textToLinkOnEnter(schema))
+  }
```

Comment:
Hi @sampullman 

I think the reason why Enter is not working in our custom keymap is because [Enter](https://github.com/ProseMirror/prosemirror-commands/blob/2da5f6621ab684b5b3b2a2982b8f91d293d4a582/src/commands.ts#L711C12-L711C25) is already handled in the baseKeymap from `prosemirror-commands`. It returns true in most cases, so any keymap added after that will not get executed.

Here's a list of all keys handled in the base keymap:

![Screenshot 2023-12-22 at 5 26 16 PM](https://github.com/samatechtw/pubstudio-builder/assets/59676941/e563db1e-d4d7-45f3-9513-3a11a55b91ff)

Reply:
Yeah, that makes sense. So to have our own custom handler, it's probably easiest to copy/paste from `[prosemirror-keymap](https://www.npmjs.com/package/prosemirror-keymap)` (or wherever it's defined), and modify it to include our own code.